packages/office-ui-fabric-react/src/components/ActivityItem/ActivityItem.tsx

```diff
@@ -0,0 +1,171 @@
+/* tslint:disable */
+import * as React from 'react';
+/* tslint:enable */
+
+import { autobind, BaseComponent, memoize } from '../../Utilities';
+import { IActivityItemProps, IActivityItemStyles } from './ActivityItem.Props';
+import { mergeStyles } from '../../Styling';
+import { getStyles } from './ActivityItem.styles';
+import { Persona, PersonaSize } from 'office-ui-fabric-react/lib/Persona';
+import { IPersonaProps } from 'office-ui-fabric-react/lib/Persona';
+
+export interface IActivityItemClassNames {
+  root?: string;
+  activityContent?: string;
+  activityText?: string;
+  personaContainer?: string;
+  activityPersona?: string;
+  activityTypeIcon?: string;
+  commentText?: string;
+  timeStamp?: string;
+}
+
+export class ActivityItem extends BaseComponent<IActivityItemProps, {}> {
+  private _classNames: IActivityItemClassNames;
+  private _styles: IActivityItemStyles;
+
+  constructor(props: IActivityItemProps) {
+    super(props);
+  }
+
+  public render() {
+    let {
+      className,
+      onRenderIcon = this._onRenderIcon,
+      onRenderActivityDescription = this._onRenderActivityDescription,
+      onRenderComments = this._onRenderComments,
+      onRenderTimeStamp = this._onRenderTimeStamp,
+      styles: customStyles,
+    } = this.props;
+
+    this._styles = getStyles(undefined, customStyles);
+    this._classNames = this._getClassNames(
+      this._styles,
+      this.props.className,
+      this.props.activityPersonas,
+      this.props.isCompact
+    );
+
+    return (
+      <div className={ this._classNames.root } style={ this.props.style } >
+
+        { this.props.onRenderIcon ?
```

Comment:
this should be `onRenderIcon(props, this._onRenderIcon)` and move that className down into the local render function